components/bpmn/bpmn-explorer-web/src/web/model/advancedFilterModel.jag

```diff
@@ -0,0 +1,237 @@
+<%/*<!--
+ ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ -->*/%>
+<%
+
+var headers = {"Authorization":"Basic " + auth};
+
+var deployedProcessCount = get( httpUrl +"/bpmn/stats/processTaskServices/deployedProcessCount" ,null, headers, "json" ).data.data;
+
+var list = [];
+
+for (var i= 0; i < deployedProcessCount.length; i++) {
+    var processDefinitionId = deployedProcessCount[i].processDefinitionId;
+    var instanceList = {
+        deployment: processDefinitionId       
+    };
+
+    list.push(instanceList);
+    
+}
+//Advanced Filter  Model
+
+var queryReceived = false; //At first request (or without query parameters) no need to load task list
+var onclickUrl = 'processMonitoring'; 
+
+if (request.getQueryString() != null) {
+
+    //If query received need to process query
+    queryReceived = true;
+    var pagination = request.getParameter("pagination");    
+    var requestParameters = request.getAllParameters();
+    var newQueryString = ""; //query string used for pagination
+    var pageSize = 10;//page size (currently it's hard coded to 10 untill we make it configurable)
+    var searchActiveInstances = true;//state that the user is searching for active tasks or completed historic tasks
+    var requestUrl = '';//BPS BPMN REST API call url
+
+    if (log.isDebugEnabled()) {
+        log.debug("Request parameters:" +stringify(requestParameters));
+        log.debug("Request parameters:" +request.getQueryString());
+    }
+
+    //set pagination
+    if (pagination == null || pagination == "") {
+        pagination = 1;
+    } else {
+        pagination = parseInt(pagination);
+    }
+
+    //remove pagiantion parameter if exists
+    if (request.getParameter("pagination") !== null) {
+        //split query string with "&" to remove pagination part from the query string
+        var qs = request.getQueryString();
+        newQueryString = qs.substring( qs.indexOf('&') + 1 );
+    } else {
+        newQueryString = request.getQueryString();
+    }
+
+    var bodyOfRequest = {};
+    var vCollection = {};
+
+    if (request.getParameter("instanceStatus") === "COMPLETED") {
+
+        searchActiveInstances = false;              
+    
+
+        if (requestParameters.processInstanceId !== undefined && requestParameters.processInstanceId.length > 0) {
```

Comment:
As a practice don't keep 2 new lines. If you want to have a new line to separate code blocks, just use a single line